### ISSUE TITLE
When I run the application, it doesn't go past the preload page. It also happens in the demo application

### ISSUE THREAD

**Issue by factorx22** [cipchk/ng-alain#2136]
### Reproduction link
https://stackblitz.com/edit/ng-alain-setup-1jsz3m?

### Steps to reproduce
ng new my-project --style less --routing
cd my-project
ng add ng-alain
npm start
open link in browser chrome, mozilla, edge

### What is expected?
To see main page after preload

### What is actually happening?
does not happen

| Environment | Info |
|---|---|
| ng-alain | 12.3.0 |
| Browser | Chrome, M Edge, Mozilla ( on windows ) |

---
even this happens with the demo https://ng-alain.surge.sh/

Console say this error: 

main.065731009467f1c06bdc.js:1 ERROR TypeError: Cannot read properties of undefined (reading 'ng')
    at Hn.use (main.065731009467f1c06bdc.js:7)
    at r.project (main.065731009467f1c06bdc.js:7)
    at r._next (main.065731009467f1c06bdc.js:7)
    at r.next (main.065731009467f1c06bdc.js:7)
    at r._next (main.065731009467f1c06bdc.js:7)
    at r.next (main.065731009467f1c06bdc.js:7)
    at v.checkIterators (main.065731009467f1c06bdc.js:7)
    at D.notifyNext (main.065731009467f1c06bdc.js:7)
    at r._next (main.065731009467f1c06bdc.js:7)
    at r.next (main.065731009467f1c06bdc.js:7)

**Comment by cipchk** [comment]
@factorx22  I can’t reproduce it, even through ng-alain.surge.sh, png:
![image](https://user-images.githubusercontent.com/2987467/142352868-6cf800ed-56c0-4342-a2c8-1daf736eb7a2.png)

**Comment by factorx22** [comment]
i thing is something about localization

**Comment by factorx22** [comment]
I have investigated a bit and found a problem when loading the StartupService.load () when running the application.

The problem is in i18n.service.ts in constructor code:

`   if (this._langs.findIndex(w => w.code === defaultLang)) {
      this._defaultLang = defaultLang;
    }
   `

This always assigns defaultLang to this._defaultLag, even if the browser's location code is not in the constant LANGS.

When there is no location file that matches the browser in \assets\tmp\i18n, the application reports the error


![image](https://user-images.githubusercontent.com/18436745/142508847-cc06f5e1-2fec-4ff9-a10b-e22cff935b67.png)

A possible correction would be:

`    if (this._langs.findIndex(w => w.code === defaultLang) > -1) {
      this._defaultLang = defaultLang;
    }`

In this way, if there is no location file, the default is taken:  `const DEFAULT = 'zh-CN';`

However the original problem I reported persists, it does not go past the preload screen. I assume it is because of the localization error, but at this point it is not redirecting to the screen that shows the error notification.

![image](https://user-images.githubusercontent.com/18436745/142512785-bb466926-de3a-4072-aeb5-629a74b6b6c5.png)

However if I debug the application in vscode it redirects well to the screen that reports the error.
